src/openrct2/management/NewsItem.h

```diff
@@ -291,7 +292,7 @@ namespace News
 
     std::optional<CoordsXYZ> GetSubjectLocation(News::ItemType type, int32_t subject);
 
-    News::Item* AddItemToQueue(News::ItemType type, rct_string_id string_id, uint32_t assoc);
+    News::Item* AddItemToQueue(News::ItemType type, rct_string_id string_id, uint32_t assoc, const Formatter& formatter);
```

Comment:
```suggestion
    News::Item* AddItemToQueue(News::ItemType type, rct_string_id string_id, uint32_t assoc, const Formatter& formatter = Formatter());
```

I guess you need this for now, unless you expect to convert them all into this commit

Reply:
Forgot to amend, I did indeed plan to have all the news use it correctly.